pages/configuration/elasticsearch.rst

```diff
@@ -119,7 +88,7 @@ system defaults allow. **Set it to at least 64000 open file descriptors.**
 
 Graylog will show a notification in the web interface when there is a node in the Elasticsearch cluster which has a too low open file limit.
 
-Read about how to raise the open file limit in the corresponding `Elasticsearch documentation page <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__.
+Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__ / `5.x <https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html>`__ documentation pages.
```

Comment:
This should be a link to the specific version of the Elasticsearch documentation:
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/file-descriptors.html